src/Tests.Microsoft.Security.Utilities.Core/IdentifiableSecretsTests.cs

```diff
@@ -792,14 +792,19 @@ public void IdentifiableSecrets_DerivedSymmetricKeys()
                     {
                         foreach (string signature in identifiablePattern.Signatures)
                         {
-                            if (IdentifiableSecrets.TryValidateBase64Key(securityKey, checksumSeed, signature, identifiablePattern.EncodeForUrl))
+                            string standaloneSecret =
```

Comment:
>standaloneSecret [](http://example.com/codeflow?start=35&length=16)

When we produce an example, it sometimes but not always comprises a standalone key. Here is a simple pattern for obtain the secret, we grab the output of the <refine> group.

I put a second pattern for doing this elsewhere just to compare.